app/models/mechanic.rb

```diff
@@ -0,0 +1,6 @@
+class Mechanic < ApplicationRecord
+  belongs_to :amusement_park
+  has_many :mechanic_rides
+  has_many :rides
+  has_many :rides, through: :mechanic_rides
```

Comment:
Ok so it does look like you added a connection between a mechanic and an amusement park. This unfortunately is incorrect and not necessary. Let's take a look at the setup instructions for the db.

These 2 entities have a one-to-many relationship with each other:
```
* AmusementParks have many Rides (has_many :rides)
* Rides belong to an AmusementPark (belongs_to :amusement_park)
```

These 2 entities have a many-to-many relationship with each other so you need a joins table between them. (ie. `MechanicRides`.
```
* Mechanics can work on many Rides (has_many :mechanic_rides / has_many :rides, through: :mechanic_rides)
* Rides can have many Mechanics working on them (has_many :mechanic_rides / has_many :mechanics, through: :mechanic_rides)
```

Any other extra associations would need to be created with a `through` relationship.

The most important part of any IC or project is going to be the database setup and if you get stuck on that then I ask that you reach out.